pyproject.toml

```diff
@@ -108,6 +108,7 @@ pycryptodome = "*"
 PyJWT = "*"
 pyserial = "*"
 pyzmq = "*"
+rerun-sdk = "*"
```

Comment:
I think we need to pin to the version from `WHEEL_URL`. don't exactly remember why though

Reply:
> A wheel is a type of [built distribution](https://packaging.python.org/glossary/#term-built-distribution). In this case, built means that the wheel comes in a ready-to-install format and allows you to skip the build stage required with source distributions.

A different way to install packages but more efficiently. This could be good to look into to improve build CI.

Reply:
I pinned to 0.16.0 since that's the current version.

https://build.rerun.io/commit/660463d/wheels/index.html